README.md

```diff
@@ -706,6 +706,8 @@ arXiv, 2020.
 - <a name="GantierCadena21tip"></a>Gantier Cadena, P. R., Qian, Y., Wang, C., Yang, M.,  
 *[SPADE-E2VID: Spatially-Adaptive Denormalization for Event-Based Video Reconstruction](https://doi.org/10.1109/TIP.2021.3052070)*,  
 IEEE Trans. Image Processing, 30:2488-2500, 2021. [Project page](https://github.com/RodrigoGantier/SPADE_E2VID)
+- [Baldwin R.W., Ruixu L., Almatrafi M., Asari V., Hirakawa K.](#Baldwin21arxiv),  
```

Comment:
We use only one last name for "pointers" (2nd categories). See Barua et al, example.

Reply:
ok, making the change